config/environments/development.rb

```diff
@@ -63,7 +63,7 @@
     if env['action_dispatch.original_path']&.include?('api/v1')
       Api::V1::ExceptionsController.action(:render_error).call(env)
     else
-      ActionDispatch::PublicExceptions.new(Rails.public_path)
+      ActionDispatch::PublicExceptions.new(Rails.public_path).call(env)
```

Comment:
is this just a cleanup bugfix?

Reply:
Actually I backed this out because it doesn't do what I wanted it to.  There's weirdness with some errors (like the `ActionDispatch::RoutingError` we say in Sentry), but most other errors are still rendered with the nice development stack trace page.